gui/src/installer/view.rs

```diff
@@ -1411,8 +1412,15 @@ pub fn edit_key_modal<'a>(
                         Column::new()
                     },
                 )
+                .push_maybe(
+                    if duplicate_master_fg {
+                        Some(text("This path already contains a key with the same master fingerprint").style(color::RED))
```

Comment:
![image](https://github.com/wizardsardine/liana/assets/22457751/c21facb8-6d76-480e-8516-d07413c44401)

Since we ask them to "select a signing device", i think we should use the same wording in the error message:
```suggestion
                        Some(text("A single signing device may not be used more than once per path. (It can still be used in other paths.)").style(color::RED))
```

I also explicitly they can still use the same signing device more than once in the whole descriptor, as it seems to have been a point of confusion in our Telegram.